### ISSUE TITLE
SC 3.1.6 is nonsensical

### ISSUE THREAD

**Issue by JaninaSajka** [w3c/wcag#1746]
The only way to read [3.1.6 Pronunciation](https://www.w3.org/TR/WCAG22/#pronunciation) as a complete sentence is to intrepret as a declaration that a mechanism is indeed available to disambiguate pronunciation. But, if that's true, why is that mechanism not indicated? And whi is this an AAA criterion? In fact this SC is a sentence fragment. No mechanism is available to be pointed to. It seems the problem is that a mechanism is wanting. The case for the importance of pronunciation is made well, but the solution won't be available until and unless the APA's Pronunciation Task Force succeeds in defining one that can be widely adopted in HTML.

**Comment by patrickhlauke** [comment]
There are a dozen or so other SCs that use the "a mechanism is available" wording, so this is nothing new unless I'm missing something? It's explicitly handwavy in the normative text because there are many different approaches to satisfy the high-level requirement, and not all of them may be down to authors themselves. The understanding https://www.w3.org/WAI/WCAG22/Understanding/pronunciation.html does provide some low-tech examples?

**Comment by patrickhlauke** [comment]
And I'd hazard a guess that it's set to AAA because this can potentially result (if done by authors for a large number of words) in quite disruptive/involved changes that need to be made to content (adding some form of pronunciation guide in brackets for a large number of words in a page, for instance).

**Comment by alastc** [comment]
Hi Janina.

> why is that mechanism not indicated? 

To allow for it to be either user-agent side or part of the authored-content. I suspect there was an expectation (or hope) that the user-agent and/or the mark-up available would improve.

> why is this an AAA criterion?

I wasn't there at the time, but I would guess because it may be difficult or impossible to apply to all content, and/or due to the level of effort required (e.g. manually tagging every potentially ambiguous word), and/or because it could make the interface worse for other audiences (as Patrick mentioned).

Wording wise I would probably add another "the" before "meaning of the words", but you seem to have understood the thrust of the SC.

**Comment by JaninaSajka** [comment]
Patrick, Alastair: I certainly agree with AAA. I also agree this guideline needs to be "hand wavy." That's the problem. It's not at all "hand wavy." It makes a false declaration when it says: "A primary mechanism exists ..." The verb is singular, and the adjective reinforces that, and it's just no honest--and that's not helpful to anyone trying to do their best by this important guideline with no technical standard to glom onto. Also, that's a long way from "a mechanism is available," which would be far more acceptable, though still not really accurate. The sad truth is that none of the work-arounds described by Understanding WCAG can be relied on across operating environments, browers and TTS engines as the [Pronunciation Task Force Gap Analysis](https://www.w3.org/TR/pronunciation-gap-analysis-and-use-cases/) demonstrates. Isn't there some way to be more honest with this guideline?

**Comment by patrickhlauke** [comment]
> It makes a false declaration when it says: "A primary mechanism exists ..." The verb is singular, and the adjective reinforces that, and it's just no honest

I'm sorry, but where does it say "A primary mechanism exists"? That wording is not present in either the normative wording of the SC, nor the understanding document.

> Also, that's a long way from "a mechanism is available," which would be far more acceptable

that *is* in fact what the SC uses/says.

**Comment by fstrr** [comment]
This issue looks like it can be closed. If it needs to be re-opened, please do that and convert it to a Discussion.
